go/libkb/sig_chain.go

```diff
@@ -646,11 +645,10 @@ func (sc *SigChain) verifySigsAndComputeKeysCurrent(ctx context.Context, eldest
 		return cached, 0, err
 	}
 
-	if sc.allKeys || sc.loadedFromLinkOne {
-		if first := sc.getFirstSeqno(); first > keybase1.Seqno(1) {
-			err = ChainLinkWrongSeqnoError{fmt.Sprintf("Wanted a chain from seqno=1, but got seqno=%d", first)}
-			return cached, 0, err
-		}
+	// AllKeys mode is now the default.
+	if first := sc.getFirstSeqno(); first > keybase1.Seqno(1) {
```

Comment:
Just checked by manually deleting some old links, and I think we're fine. The LoadFromStorage step will short circuit and return `nil` if any of the links we're trying to load are missing ([here](https://github.com/keybase/client/blob/e97bab9/go/libkb/sig_chain.go#L914) and [here](https://github.com/keybase/client/blob/e97bab9/go/libkb/sig_chain.go#L935)), which will cause a new server fetch of the entire chain.